src/main/java/com/whitesource/remediation/encoder/Utils.java

```diff
@@ -0,0 +1,11 @@
+package com.whitesource.remediation.encoder;
+
+import org.owasp.esapi.ESAPI;
+import org.owasp.esapi.codecs.AbstractCharacterCodec;
+
+public class Utils {
```

Comment:
A util class just for this is in my view reduces readability so it should be removed.